include/fizzy/fizzy.h

```diff
@@ -159,10 +159,10 @@ void fizzy_free_module(const FizzyModule* module);
 
 /// Get type of the function defined in the module.
 ///
-/// @param module   Pointer to module. Cannot be NULL.
-/// @param func_idx Function index. Can be either index of an imported function or of a function
-///                 defined in module. Behaviour is undefined, if index is not valid according to
-///                 module definition.
+/// @param module      Pointer to module. Cannot be NULL.
+/// @param func_idx    Function index. Can be either index of an imported function or of a function
```

Comment:
IMHO it is fine to just keep <=2 spaces between param name and description.

Reply:
I agree, vote for 2 spaces.

Also https://github.com/wasmx/fizzy/pull/664#discussion_r529725849 ^^

Reply:
Changed both widths.

Reply:
Oh sorry, I meant the 2 spaces for between `@param` and `module`, and 4 spaces between `module `and `Pointer to module` 😬

Reply:
What is the point of additional fucking spaces?

Reply:
It was the style @gumb0 used half time and I found it rather readable.

Reply:
I think it looks more readable when it's more spaced out.

(what I originally used was starting comments on tab positions)